packages/dev/core/src/Materials/materialHelper.functions.ts

```diff
@@ -64,19 +70,38 @@ export function BindFogParameters(scene: Scene, mesh?: AbstractMesh, effect?: Ef
  * @param attribs The current list of supported attribs
  * @param mesh The mesh to prepare the morph targets attributes for
  * @param influencers The number of influencers
+ * @param usePositionMorph Whether the position morph target is used
+ * @param useNormalMorph Whether the normal morph target is used
+ * @param useTangentMorph Whether the tangent morph target is used
+ * @param useUVMorph Whether the UV morph target is used
+ * @param useUV2Morph Whether the UV2 morph target is used
  */
-export function PrepareAttributesForMorphTargetsInfluencers(attribs: string[], mesh: AbstractMesh, influencers: number): void {
+export function PrepareAttributesForMorphTargetsInfluencers(
+    attribs: string[],
+    mesh: AbstractMesh,
+    influencers: number,
+    usePositionMorph = true,
+    useNormalMorph = false,
+    useTangentMorph = false,
+    useUVMorph = false,
+    useUV2Morph = false

```

Comment:
There are five more parameters. I'm mentioning this because it's a public API and we might need to design it more carefully.